tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -164,6 +164,16 @@ describe('Acceptance: addon-smoke-test', function() {
       });
   });
 
+  it('ember addon with linting errors', function() {
+    return copyFixtureFiles('addon/with-linting-errors')
+      .then(function() {
+        return ember(['test']);
+      })
+      .then(function(result) {
+        expect(result.exitCode).to.not.eql(0);
```

Comment:
Can we also check the `results.output` to confirm that the right test failed? Something like (stolen from another test I recently added):

```
var output = result.output.join(EOL);
expect(output).to.match(/TemplateLint:/, 'ran template linter');
```